biodatacatalyst-ui/src/main/webapp/picsureui/search-interface/tag-filter-view.js

```diff
@@ -221,7 +221,7 @@ function(BB, HBS, _, tagFilterViewTemplate, tagFilterModel, filterModel, keyboar
 		render: function(){
 			let unusedTags = this.model.get("unusedTags").toArray();
 			let filteredTags = _.filter(unusedTags, function(tag){
-				return ! (tag.get('tag')===dccHarmonizedTag || studyVersionRegex.test(tag.get('tag'))) && ! tableVersionRegex.test(tag.get('tag'));
+				return ! (tag.get('tag')===dccHarmonizedTag || searchUtil.isStudy(tag.get('tag'))) && ! tableVersionRegex.test(tag.get('tag'));
```

Comment:
Did you mean for this to be included in the pr? You have it in this one already: https://github.com/hms-dbmi/pic-sure-bdc-frontend/pull/270/files